docs/docs/forms.md

```diff
@@ -186,7 +186,7 @@ Overall, this makes it so that `<input type="text">`, `<textarea>`, and `<select
 
 When you need to handle multiple controlled `input` elements, you can add a `name` attribute to each element and let a handler function choose what to do based on the value of `event.target.name`. For example:
 
-```javascript{14-15,18,32,39}
```

Comment:
I've already updated the Codepen! Is there anything specific that needs improvement? :)